core/directory.go

```diff
@@ -37,6 +37,11 @@ type Directory struct {
 
 	// Services necessary to provision the directory.
 	Services ServiceBindings
+
+	// The list of excluded paths of that directory.
+	// This is useful for the ignore directive to avoid double exclusion between
+	// CLI call and module call.
+	Excluded []string
```

Comment:
I'm not sure how to use this yet, I keep a list of ignored pattern but I'm not sure it's worth it.

There's no more `resolveDirectoryFromCaller` so I need an hint on that.